mod/mod.go

```diff
@@ -13,7 +13,7 @@ import (
 )
 
 const (
-	Dependency = "go-mod"
+	Dependency = "go-mod-vendor"
```

Comment:
@sophiewigmore but it does look like it was changed, re: https://github.com/paketo-buildpacks/go-mod-vendor/pull/72/files#r454453176